examples/java/src/main/java/org/apache/beam/examples/complete/game/HourlyTeamScore.java

```diff
@@ -170,7 +176,7 @@ public static void main(String[] args) throws Exception {
         // Extract and sum teamname/score pairs from the event data.
         .apply("ExtractTeamScore", new ExtractAndSumScore("team"))
         .apply(
-            "WriteTeamScoreSums", new WriteToText<>(options.getOutput(), configureOutput(), true));
+            "WriteTeamScoreSums", new WriteToText<>(options.getOutput(), configureOutput(), false));
```

Comment:
This is just to write the results to one file per round instead of several files per windowing. The windowing itself is not being changed if I'm not mistaken.
I added an extra option and set it to true as default to keep the original functionality.